AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,8 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+nurgsm.com##div[id^="sgpb-popup-dialog"]
+nurgsm.com##div[class$="sgpb-popup-overlay"]
```

Comment:
```
nurgsm.com##.sgpb-popup-overlay
nurgsm.com##.sgpb-popup-dialog-main-div-wrapper
```
Why not? Also duplicate these rules to the Social - `popups.txt`

Reply:
Aren't the rules too big then?

Reply:
For now, it's ok.